tests/jobs/test_db.py

```diff
@@ -1,9 +1,8 @@
-from virtool_core.models.job import Job
-
 from virtool.jobs.db import fetch_complete_job
 
-async def test_fetch_complete_job(snapshot, mongo, fake2, static_time, test_job):
+
+async def test_fetch_complete_job(snapshot, fake2, mongo, static_time, test_job):
     user = await fake2.users.create()
     test_job["user"] = {"id": user.id}
 
-    assert await fetch_complete_job(mongo, test_job) == snapshot
+    assert repr(await fetch_complete_job(mongo, test_job)) == snapshot
```

Comment:
Why are you bothering to use `repr`? Wasn't it working without?

Reply:
Without using repr, it doesn't include the class names which are in the generated snapshot